src/common/cloudinary.ts

```diff
@@ -53,3 +53,35 @@ const avatarPreset = 'avatar';
 
 export const uploadAvatar = (userId: string, stream: Readable) =>
   uploadFile(`${avatarPreset}_${userId}`, avatarPreset, stream);
+
+const postPreset = 'post_image';
+
+export const uploadPostFile = (
+  name: string,
+  stream: Readable,
+): Promise<string> =>
+  new Promise((resolve, reject) => {
+    const outStream = cloudinary.v2.uploader.upload_stream(
+      {
+        public_id: name,
+        upload_preset: postPreset,
+      },
+      (err, callResult) => {
+        if (err) {
+          return reject(err);
+        }
+
+        return resolve(
+          cloudinary.v2.url(callResult.public_id, {
+            secure: true,
+            fetch_format: 'auto',
+            quality: 'auto',
+          }),
+        );
+      },
+    );
+    stream.pipe(outStream);
+  });
+
+export const uploadPostImage = (postId: string, stream: Readable) =>
+  uploadPostFile(`${postPreset}_${postId}`, stream);
```

Comment:
`postId` should be enough as a name I believe.
One more question, we should support adding images to the content of the post not only the cover image. Is it covered here or in a different PR?

Reply:
It would be covered in the iteration of v6 where we will now allow uploading image content. As part of v5, it should not be the case yet. So in v6 we will introduce a new mutation for that case.